Makefile

```diff
@@ -78,3 +83,38 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
+
+# Run golangci-lint
+lint:
+	golangci-lint run ./... -v
+
+# Verify go dependency and generated manifest files
+static-test: generate fmt vet manifests verify
+	git diff --exit-code
+
+# Run unit test
+unit-test:
+	go test -v ./controllers/... -ginkgo.v -ginkgo.failFast
+
+# Minikube related command
+minikube-download:
+	curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+	chmod +x minikube
+	sudo mkdir -p /usr/local/bin/
+	sudo install minikube /usr/local/bin/
+	sudo snap install kubectl --classic
+	sudo apt-get install conntrack
+
+minikube-start:
+	CHANGE_MINIKUBE_NONE_USER=true sudo -E minikube start --driver=none --kubernetes-version=v1.19.7
+	sleep 3
+
+minikube-clean:
+	CHANGE_MINIKUBE_NONE_USER=true sudo -E minikube delete
+
+# Kubebuilder related command
+kubebuilder-download:
+	curl -L https://go.kubebuilder.io/dl/2.3.1/linux/amd64 | tar -xz -C /tmp/
+	sudo mv /tmp/kubebuilder_2.3.1_linux_amd64 /usr/local/kubebuilder
+	export PATH=$(PATH):/usr/local/kubebuilder/bin
+
```

Comment:
Please add `make clean` to cleanup after `make deploy` command.
```
clean:
	kustomize build config/default | kubectl delete -f -
```

Reply:
Added :D